models/errors_whitebox_test.go

```diff
@@ -0,0 +1,30 @@
+package models
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/almighty/almighty-core/resource"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSimpleError_Error(t *testing.T) {
+	t.Parallel()
+	resource.Require(t, resource.UnitTest)
+	e := simpleError{message: "foo"}
+	assert.Equal(t, "foo", e.Error())
+}
+
+func TestBadParameterError_Error(t *testing.T) {
+	t.Parallel()
+	resource.Require(t, resource.UnitTest)
+	e := BadParameterError{parameter: "foo", value: "bar"}
+	assert.Equal(t, fmt.Sprintf(stBadParameterErrorMsg, e.parameter, e.value), e.Error())
+}
+
+func TestNotFoundError_Error(t *testing.T) {
+	t.Parallel()
+	resource.Require(t, resource.UnitTest)
+	e := NotFoundError{entity: "foo", ID: "bar"}
+	assert.Equal(t, fmt.Sprintf(stNotFoundErrorMsg, e.entity, e.ID), e.Error())
```

Comment:
Sure it does. It's a struct with functions on it. But no matter what the opinion is, the coverage output sees logic in there. Let's see if we run into troubles when updating the errors and talk about the concerns then, okay?